public/app/features/dashboard/components/GenAI/utils.ts

```diff
@@ -53,13 +54,13 @@ export function getDashboardChanges(dashboard: DashboardModel): {
 }
 
 /**
- * Check if the LLM plugin is enabled and configured.
- * @returns true if the LLM plugin is enabled and configured.
+ * Check if the LLM plugin is enabled.
+ * @returns true if the LLM plugin is enabled.
  */
 export async function isLLMPluginEnabled() {
-  // Check if the LLM plugin is enabled and configured.
+  // Check if the LLM plugin is enabled.
   // If not, we won't be able to make requests, so return early.
-  return await openai.enabled();
+  return llms.openai.enabled().then((response) => response.ok);
```

Comment:
Not an issue to address with this PR but it seems that we aren't currently using the `configured` boolean of the `enabled` API (see [here for docs on what this endpoint returns](https://github.com/grafana/grafana-experimental/blob/main/src/llms/types.ts#L12))

We may want to consider handling the case where LLM plugin is installed is not configured yet so "point" user in right direction / docs to finish the configuration? (maybe have buttons show up but disabled with a link to plugin settings / docs?

Reply:
Yep I think the new health check should provide much more detail about what the user needs to do to get the features working, you should be able to use the `error` field for that!